docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -12,17 +12,18 @@ filter to be kept separate from any keys used by other systems. This document as
 will be to prefix the alias used by the filter with the word `KEK_`. If a different naming convention is used, adapt
 the instructions accordingly.
 
-== Administrator Actor
+== Role of the Administrator
 
-To use the filter, there must be an administrative identity established within AWS IAM.  This user, which is likely to be a human,
-has the responsibility to manage keys and aliases within AWS KMS for use by the filter.
+In order to use the filter, there must be an *Administrator*.  The Administrator has the responsibility
+to create encryption keys within AWS KMS which will be used to encrypt the records. It is up to the organization
+deploying Record Encryption to decide how the role of the *Administrator* maps into their organization structure.
```

Comment:
```suggestion
In order to use the filter, there must be an external entity which is responsible for creating encryption keys within AWS KMS which will be used to encrypt the records. It is up to the organization
deploying Record Encryption to decide how this entity is managed.
```

I worry a bit that talking about an Administrator and mapping to the org structure assumes a human rather than some cloud formation or terraform code to manage it via git ops. 

That said talking about Administrators and org structures maybe makes our point more intelligible.